src/views/link.js

```diff
@@ -4,7 +4,7 @@ import c from 'classnames'
 export default class Link extends Component {
   render (props, _, { theme }) {
     const {
-      color = theme.mainColor,
+      color = window.matchMedia('(prefers-color-scheme: dark)').matches ? theme.headerColor : theme.mainColor,
```

Comment:
This won't work with server-side rendering since `window` doesn't exist on the server. We can fix this by using `store.app.colorScheme` which is either `'light'` or `'dark'`. I'll fix it in a followup commit.